OpenSDS_Volume_Actions_Design.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Status: Pending
+
+Version: Beta
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+Extends the size of a volume. 
+
+## Motivation
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Goals
+
+What will be addressed by the proposal?
```

Comment:
I think it's an explanation to explain the meaning of `Goals`, please remove all explanation of this document